pages/whats-new/product-roadmap.md

```diff
@@ -30,11 +30,11 @@ every few months with the status of our progress, as well as add new
 high-level future requests and ideas.
 
 {% for milestone in site.data.milestones %}
-<section>
+<section id="{{ milestone.id }}-section">
   <h2 id="{{ milestone.id }}">{{ milestone.title }}</h2>
   <ul>
   {% for task in milestone.tasks %}
-    <li>
+    <li id="tooltip-text-{{ task.title | slugify }}">
```

Comment:
@donjo the only error that's flagging for me at the moment is:
```
'http://localhost:4000/about-our-work/product-roadmap/ | critical | ARIA attributes must conform to valid values | ["#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(1) > .usa-label.label-done", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(2) > .usa-label.label-in-progress", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(3) > .usa-label.label-in-progress", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(4) > .usa-label.label-in-progress", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(5) > .usa-label.label-in-progress", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(7) > .usa-label.label-in-progress", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(8) > .usa-label.label-in-progress", "#main-content > .styleguide-content.usa-content > section:nth-of-type(1) > ul > li:nth-of-type(9) > .usa-label.label-in-progress"]'
```
The `aria-describedby="tooltip-text-x"` was present when each of these labels had tooltips, but they no longer have tooltips, so I'm wondering if it's still needed or not.

Reply:
Just checked with @nickbristow and said this is good, so I think it's good to leave it in.